### PR TITLE
Docs-1514-3.17.1-GA

### PR DESCRIPTION
**Pull request by bmckercher123**
Product Version(s):
Calico Enterprise

Issue:
https://tigera.atlassian.net/browse/DOCS-1514

Link to docs preview:

**Dev/QE/PM**
- [x] Signoff - release.json updates to Tigera operator, chart version, component versions
- [x] Review RN Known Issues (add/remove)

**Writer**
- [x] Verify install manifests after Eng merge including helm charts
- [x] RN
     - [x] Verify that all versions are showing in the component list
    - [x] Update date
    - [x] Check archived link
- [x] Edit and grammar check RN
- [x] Verify version drop-down
- [x] Verify archived doc links

**Merge checklist**
- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

**Comment by bmckercher123** [comment]
From @danudey...We'll end up using:
- Operator v1.30.4
- key-cert-provisioner v1.1.10

**Review by danudey — APPROVED** [review]
lgtm

File: calico-enterprise/operations/cnx/configure-identity-provider.mdx [hunk 1]
```diff
@@ -248,6 +248,7 @@ When configuring your cluster, you may be asked for the following inputs:
      bindPW: <your-base64-bind-password>
      rootCA: <your-base64-ca-cert-pem>
    ```
+###
```
Comment (hjiawei) [inline calico-enterprise/operations/cnx/configure-identity-provider.mdx:251]: Do we need this change?

File: calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.1-release-notes.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+08 August 2023
+
+### What's new {#whats-new-3.17.1}
+
+#### New policy recommendations engine for namespace isolation
+{{prodname}} has added a new policy recommendations engine that automatically generates staged policies for namespace isolation within your cluster. [Policy recommendations](network-policy/recommendations/policy-recommendations.mdx).
```
Comment (hjiawei) [inline calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.1-release-notes.mdx]: I believe `mdx` files are transpiled to web pages. You need to link to `network-policy/recommendations/policy-recommendations` (not `.mdx`). Please fix all other `mdx` references in this release notes.

Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.1-release-notes.mdx]: Fixed.

File: calico-enterprise/operations/cnx/configure-identity-provider.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico-enterprise/operations/cnx/configure-identity-provider.mdx:251]: It looks like a variable where user inserts their own value.

**Comment by hjiawei** [comment]
@bmckercher123 Looks like after this merge, Calico Enterprise v3.17 is latest but also gives a page not found error?

Screenshot from 2023-08-05 21-24-36

Here are the steps to reproduce:
1. go to https://docs.tigera.io/
2. click "Calico Enterprise" doc set in the middle.

**Pull request merged**